### PR TITLE
feat: pressing enter with multiple blocks should create block

### PR DESCRIPTION
**Pull request by zqran**
close: #1378

### CODE REVIEW

File: packages/blocks/src/page-block/utils/bind-hotkey.ts [hunk 1]
```diff
@@ -298,7 +298,35 @@ export function bindHotkeys(
       selection.clear();
       return;
     }
-    // TODO fix native selection enter
+    if (blockRange.type === 'Native') {
+      e.stopPropagation();
+      e.preventDefault();
```
Comment (lawvs) [inline packages/blocks/src/page-block/utils/bind-hotkey.ts:303]: Is it necessary?

Comment (zqran) [reply packages/blocks/src/page-block/utils/bind-hotkey.ts]: i'll fix it

**Review by lawvs — COMMENTED** [review]
Could you please add some tests for it?

**Comment by zqran** [comment]
Why did the test pass locally and fail here? 🤔

**Comment by lawvs** [comment]
It looks like you break the entering behavior in the code-block, can you check it?

**Review by lawvs — APPROVED** [review]
Thank you very much for your contribution. I will handle some edge cases. Please sit back and relax.

**Pull request merged**